devtools/chain/README.md

```diff
@@ -1,8 +1,77 @@
- # The meaning of each transactions in Genesis.json
- 1. `nonce = 0` means: Creation of metadata implementation contract
- 2. `nonce = 1` means: Creation of wckb contract
- 3. `nonce = 2` means: Creation of metadata proxy contract
- 4. `nonce = 3` means: Creation of crosschain implementation contract
- 5. `nonce = 4` means: Transaction of metadata update
- 6. `nonce = 5` means: Creation of crosschain proxy contract
- 7. `nonce = 6` means: Transaction of wckb grants role to crosschain
+# Quick Start: Running Axon and Blockscan Locally with Docker
+
+Run Axon and it's explorer ([blockscan](https://github.com/Magickbase/blockscan)) locally in Docker containers using [docker-compose](https://github.com/docker/compose). Please read [the `docker-compose` file](./docker-compose.yml) to learn more about the services:
+
+- Axon node
+- Blockscan as the explorer of Axon
+- Postgres database
+
+**Note**: This integration is not production ready.
```

Comment:
**Note**: This integration is not ready for production use.

Reply:
https://github.com/axonweb3/axon/pull/1364/commits/03063698251a926b32febce5c85e50632bb64ed3